playbooks/templates/dev-install_net_config.yaml.j2

```diff
@@ -1,6 +1,9 @@
 #vi:syntax=yaml
 ---
 network_config:
+{% if network_config is defined %}
+{{ network_config | to_nice_yaml }}
```

Comment:
This would actually be better named "network_config_extra" or something. 
because what is really happening is that whatever the variable "network_config" has in default.yaml is being added to the network_config. Or is is replacing it?